.expeditor/scripts/end_to_end/run_e2e_test_core.ps1

```diff
@@ -24,7 +24,7 @@ if ($IsLinux -Or $IsMacOS) {
 # It may take a while to download a given service package, and
 # services may have long init scripts. Let's be generous in how long
 # we're willing to wait.
-$DefaultServiceTimeout = 40
+$DefaultServiceTimeout = 90
```

Comment:
Might be worth mentioning that the size of this timeout takes into account apparent Buildkite slowness, too.